### PR TITLE
Update CONTRIBUTORS.csv

### PR DESCRIPTION
**Pull request by Retamogordo**
Add Retamogordo to contributor list

### CODE REVIEW

File: CONTRIBUTORS.csv [hunk 1]
```diff
@@ -114,3 +114,4 @@ yes,Handschrift,Elias Brenigmer,<cahit.kilinc@gmx.de>
 yes,Guileas,Gwénola ETHEVE,<guileas@etheve.eu> <ethevegwenola@gmail.com>
 yes,tmuro17,Tanner Muro,<tanner@tmuro17.dev> <tmuro17@gmail.com>
 yes,edbastelli,Edward Bastelli,<edbastelli@users.noreply.github.com>
+,Retamogordo,Yury Yukhananov,<yury.yukhananov@tutanota.com>
```
Comment (mrinalwadhwa) [inline CONTRIBUTORS.csv:117]: @Retamogordo Thank you for sending this.
To indicate that you accept the CLA - could you  add the explicit `yes`

```suggestion
yes,Retamogordo,Yury Yukhananov,<yury.yukhananov@tutanota.com>
```

Comment (Retamogordo) [reply CONTRIBUTORS.csv:117]: Oh, thank you. Next time it will pass smoother, I hope :)

**Comment by mrinalwadhwa** [comment]
Thank you for accepting the CLA!!

File: CONTRIBUTORS.csv (hunk 1 above)
Comment (mrinalwadhwa) [reply CONTRIBUTORS.csv:117]: I added the yes for you 🙏

**Pull request merged**